docs/source/daml/reference/choices.rst

```diff
@@ -72,10 +72,59 @@ Controllers
 
   The conjunction of **all** the parties are required to authorize when this choice is exercised.
 
-.. _daml-ref-anytime:
+.. _daml-ref-consumability:
+
+Consumability
```

Comment:
```suggestion
Choice Consumption
```
"Consumability" doesn't give the right connotations to me, and not sure it's a real word :D

Reply:
I'm not sure it's a word either but I copied it from the existing documentation. :slightly_smiling_face: I'm not sure "choice consumption" is particularly apt either, maybe it's because I'm not a native speaker but it almost make it sound as if the object of consumption if the choice.

The consuming/non-consuming property is called "exercise kind" in the ledger model documentation. That doesn't touch on pre-/post- concerns though, so I'm not sure we want to go that way (plus, kind already has an established meaning in programming language parlance).

What about "contract consumption"?

Reply:
How about "Contract Consumption of Choices" (or "in Choices")?

Reply:
"Consumption annotations"?

Reply:
I'm saying this because this is the page on choices, I'm not sure how much sense it makes to add "choices" to the title of a paragraph. Seems redundant.

Reply:
I changed to "contract consumption" in https://github.com/digital-asset/daml/pull/4134/commits/c9961e31ced894b7cf5f2507998ad2b98bbab41c

I also changed the mention of "consumability" in `structure.rst` in https://github.com/digital-asset/daml/pull/4134/commits/11f52f472e3cb698771d149a1e0d4ac0a464f02a to reflect the new naming.

Happy to make further changes if you think this wording is still not clear enough.

Reply:
Perfect, thanks!